pyproject.toml

```diff
@@ -30,7 +30,6 @@ ci = [
   "monty[optional]",
   "pytest>=8",
   "pytest-cov>=4",
-  "types-orjson",
```

Comment:
This should be [unnecessary now](https://pypi.org/project/types-orjson/):
> Note: The orjson package includes type annotations or type stubs since version 3.6.1. Please uninstall the types-orjson package if you use this or a newer version.

[orjson 3.6.1 is released Aug 4, 2021](https://pypi.org/project/orjson/3.6.1/)